src/core/src/node.cpp

```diff
@@ -740,7 +740,8 @@ inline void update_output_tensors(ov::TensorVector& output_values, const ngraph:
             if (!tensor && outputs[i]->get_partial_shape().is_static()) {
                 tensor = ov::Tensor(outputs[i]->get_element_type(), outputs[i]->get_shape());
             }
-            output_values[i] = tensor;
+            output_values[i].set_shape(tensor.get_shape());
+            tensor.copy_to(output_values[i]);
```

Comment:
should copy_to automatically set_shape?

Reply:
Yes it can, but in case if shapes are the same it won't modify shapes.